packages/ui/components/dialog/Dialog.tsx

```diff
@@ -148,7 +148,7 @@ export function DialogFooter(props: {
       <div
         className={classNames(
           "flex justify-end space-x-2 pb-4 pt-4 rtl:space-x-reverse",
-          props.showDivider && "-mb-4"
+          !props.showDivider && "pb-8"
```

Comment:
Modal without divider should have padding bottom 32px

<img width="353" alt="Screenshot 2023-07-18 at 1 50 38 PM" src="https://github.com/calcom/cal.com/assets/53316345/49306c39-16ce-4e99-9a9a-355404aaf9e2">